### PR TITLE
Add Super Graph

### PR DESCRIPTION
**Pull request by kpbird**
https://supergraph.dev/docs/home

Fetch data without code! Super Graph gives you a high performance GraphQL API without you having to write any code. GraphQL is automagically compiled into an efficient SQL query. Use it either as a library or a standalone service.

### CODE REVIEW

**Comment by dosco** [comment]
Can you point it to just https://supergraph.dev without the `docs/home`

File: README.md [hunk 1]
```diff
@@ -311,6 +311,7 @@ If you want to contribute to this list (please do), send me a pull request.
 * [c-graphqlparser](https://github.com/tecbot/c-graphqlparser) - Go-gettable version of the libgraphqlparser C library for parsing GraphQL.
 * [tallstreet-graphql](https://github.com/tallstreet/graphql) - GraphQL parser and server for Go that leverages libgraphqlparser
 * [go-graphql](https://github.com/playlyfe/go-graphql) - A powerful GraphQL server implementation for Golang
+* [Super Graph](https://supergraph.dev/docs/home): Fetch data without code! Super Graph gives you a high performance GraphQL API without you having to write any code. GraphQL is automagically compiled into an efficient SQL query. Use it either as a library or a standalone service.
```
Comment (dosco) [inline README.md:314]: This should be https://supergraph.dev without the path `docs/home`

Comment (kpbird) [reply README.md:314]: Sure I will do it

**Comment by TreTuna** [comment]
Looks good, thank you!

**Pull request merged**